app/src/forms/submission/service.js

```diff
@@ -164,9 +164,9 @@ const service = {
         id: uuidv4(),
         submissionId: submissionId,
         submissionStatusId: data.submissionStatusId,
-        note: data.note,
+        note: data.note ? data.note : data.emailComment,
```

Comment:
We shouldn't be needing to tweak this file at all - a note is a note, no matter what context it comes from. Changing the API shape like this is bad practice.